ql/src/test/queries/clientpositive/columnstats_partlvl_dp.q

```diff
@@ -70,3 +70,9 @@ describe formatted Employee_n0 partition (employeeSalary='6000.0', country='UK')
 describe formatted Employee_n0 partition (employeeSalary='6000.0', country='UK') c;
 describe formatted Employee_n0 partition (employeeSalary='6000.0', country='UK') d;
 
+analyze table Employee_n0 partition (employeeSalary,country) compute statistics for columns;
+
+describe formatted Employee_n0 partition (employeeSalary='6000.0', country='UK') employeeName;
+describe formatted Employee_n0 partition (employeeSalary='6000.0', country='UK') c;
+describe formatted Employee_n0 partition (employeeSalary='6000.0', country='UK') d;
```

Comment:
We might want to add tests where we insert different data into partitions where only the letter cases are different to check the results, like `country='UK'` and `country='uK'`

Reply:
done